test/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,25 @@
+package codeu.chat.util;
+import codeu.chat.util.Tokenizer;
+import org.junit.Test;
+import java.io.IOException;
+import static org.junit.Assert.*;
+public final class TokenizerTest {
+	@Test
+	public void testWithNoQuotes() throws IOException {
+		final Tokenizer tokenizer = new Tokenizer("hello world how are you");
+		assertEquals(tokenizer.next(), "hello");
```

Comment:
nit: It's okay to do this for a short sentence. If you're testing with a long sentence, you can instead create an array with the expected values and loop over them.